docs/pages/api/binaries/latest.ts

```diff
@@ -0,0 +1,73 @@
+import type { NextRequest } from "next/server";
+
+const REGISTRY = "https://registry.npmjs.org";
+const SUPPORTED_PACKAGES = new Set(["turbo"]);
+const TAG = "latest";
+
+const ERRORS = {
+  NOT_FOUND: {
+    status: 404,
+    error: "Not Found",
+  },
```

Comment:
Unused?

Reply:
This was a mistake from a last second refactor, good catch! Resolved now (it's used)